app/src/main/java/org/jellyfin/androidtv/ui/playback/VideoManager.java

```diff
@@ -252,7 +252,6 @@ public void start() {
                 return;
             }
             mExoPlayer.setPlayWhenReady(true);
-            mExoPlayerView.setKeepScreenOn(true);
```

Comment:
There's still calls for "mSurfaceView.setKeepScreenOn(false);" which we probably want to remove too.

Reply:
whoops